docs/basic-operations.md

```diff
@@ -0,0 +1,105 @@
+<!--[metadata]>
++++
+title = "Basic Operations"
+description = "Basic operations using Notary"
+keywords = ["docker, notary, trust, image, signing, repository, cli"]
+[menu.main]
+parent="mn_notary"
++++
+<![end-metadata]-->
+
+# Basic operations using Notary
+
+The notary CLI provides a convenient way of managing trusted collections and the keys associated with them. On this page we describe a high level overview of the functionality of each subcommand. For a quick walk-through of the basic Notary functionality check out [getting started with Notary](gettingstarted.md).
+
+## Creating a new collection of data
+
+Before you create a new collection of data, you have to name it. In notary, each collection of data is identified by a Global Unique Name (GUN). GUNs can be arbitrary, but they have to be unique. An easy way of having unique GUNs is by including any other pre-existing global namespace already available such as DNS. A good example of a GUN is: example.com/user/collection.
+
+After chosing a name, you can initialize a repository by running the `init` command:
+
+```
+$ notary init example.com/user/collection
+```
+
+On initialization, notary creates all the necessary keys needed to sign content for this collection, create the necessary TUF metadata files and push them to the remote notary server. If you already have a root key, notary uses it instead of generating a new one.
+
+The default notary server URL configured with Notary is [https://notary-server:4443/]. This default value can overridden (by priority order):
+
+	* by specifying the option `--server/-s` on commands requiring call to the notary server.
+	* by setting the `NOTARY_SERVER_URL` environment variable.
+	* by setting the `remote_server` option inside of your configuration file (defaults to `~/.notary/config.json)
+
+## Adding and deleting content from an existing trusted collection
+
+After you initialize a trusted collection you can add and remove any content from it. You can do this by running the `add` and `remove` commands, respectively:
+
+```
+$ notary add example.com/user/collection TARGET FILEPATH
+$ notary remove example.com/user/collection TARGET
+```
+
+The `add` command takes the Global Unique Name of the trusted collection, a target name, and the path to the file you want to add.
```

Comment:
I wonder if we should specify that only one file can be added at a time?  For instance `dirname/*` gets expanded in the shell and only the first file gets added.